Makefile

```diff
@@ -348,6 +361,73 @@ docker-images-only: docker-images-cassandra \
 	docker-images-tracegen \
 	docker-images-anonymizer
 
+.PHONY: multiarch-docker
+multiarch-docker: build-binaries-linux \
+	build-binaries-s390x \
+	multiarch-docker-images-only
+
+.PHONY: multiarch-docker-images-only
+multiarch-docker-images-only: docker-images-jaeger-backend-multiarch \
+	docker-images-elastic-multiarch \
+	docker-images-tracegen-multiarch \
+	docker-images-anonymizer-multiarch
+
+.PHONY: create-baseimage-multiarch
+create-baseimage-multiarch:
+	docker buildx build -t $(BASE_IMAGE_MULTIARCH) --push \
+		--build-arg root_image=$(ROOT_IMAGE) \
+		--build-arg cert_image=$(CERT_IMAGE) \
+		--platform=$(PLATFORMS) \
+		docker/base
+	echo "Finished building multiarch base image =============="
+
+.PHONY: docker-images-jaeger-backend-multiarch
+docker-images-jaeger-backend-multiarch: create-baseimage-multiarch
+	for component in agent collector query ingester ; do \
+		docker buildx build --output "$(PUSHTAG)" \
+    		--progress=plain --target release \
+    		--build-arg base_image=$(BASE_IMAGE_MULTIARCH) \
+			--build-arg debug_image=$(GOLANG_IMAGE) \
+    		--platform=$(PLATFORMS) \
+    		--file cmd/$$component/Dockerfile \
+    		$(subst JAGERCOMP,$$component,$(IMAGE_TAGS)) \
+			cmd/$$component; \
+		echo "Finished building $$component ==============" ; \
+	done
+
+.PHONY: docker-images-elastic-multiarch
+docker-images-elastic-multiarch:
+	docker buildx build --output "$(PUSHTAG)" \
+		--progress=plain \
+		--platform=$(PLATFORMS) \
+		${ESINDEX_TAG} \
+		plugin/storage/es
+	docker buildx build --output "$(PUSHTAG)" \
+		--progress=plain \
+		--platform=$(PLATFORMS) \
+		--file plugin/storage/es/Dockerfile.rollover \
+		${ESROLLOVER_TAG} \
+		plugin/storage/es
+	@echo "Finished building multiarch jaeger-es-indices-clean =============="
+
+.PHONY: docker-images-tracegen-multiarch
+docker-images-tracegen-multiarch:
+	docker buildx build --output "$(PUSHTAG)" \
+    	--progress=plain \
+    	--platform=$(PLATFORMS) \
+    	${TRACEGEN_TAG} \
+		cmd/tracegen/
+	@echo "Finished building multiarch jaeger-tracegen =============="
+
+.PHONY: docker-images-anonymizer-multiarch
+docker-images-anonymizer-multiarch:
+	docker buildx build --output "$(PUSHTAG)" \
+    	--progress=plain \
+    	--platform=$(PLATFORMS) \
+    	$(ANONYMIZER_TAG) \
+		cmd/anonymizer/
+	@echo "Finished building multiarch jaeger-anonymizer =============="
+
 .PHONY: docker-push
```

Comment:
we should delete this target, it's not used from GH Actions, and it probably incorrect given these changes.

Reply:
OK, I've deleted this target.